opam repo add gives errors for file target where it should not

This is tested with opam 2.0.5.

When I add a local repo to some switch as follows:
```
opam repo --this-switch add devel file:$HOME/Opam/devrepo
```
this works fine. But when I later add the same repo to a different switch I get the error message:
``` 
[ERROR] Repository devel is already set up and points to file:///Users/<me>/Opam/devrepo. To change that, use
        'opam repository set-url'.
```
This is not in line with the fact that I can add a git based repo to as many switches I want with e.g.:
```
opam repo add coq-released https://coq.inria.fr/opam/released
```
Also if I leave away the file url, it works fine:
```
opam repo --this-switch add devel
```
So I would say the behaviour is unexpected since adding the repo with explicitly giving an URL is a legitimate operation and also this is inconsistent between file:// and https://.

--- a/src/client/opamRepositoryCommand.ml
+++ b/src/client/opamRepositoryCommand.ml
@@ -64,18 +64,20 @@ let add rt name url trust_anchors =
     -> rt
   | Some r ->
     OpamConsole.error_and_exit `Bad_arguments
-      "Repository %s is already set up %s. To change that, use 'opam \
-       repository set-url'."
+      "Repository %s is already set up%s. To change that, use 'opam \
+       repository set-url %s %s'."
       (OpamRepositoryName.to_string name)
       (if r.repo_url <> url then
-         "and points to "^OpamUrl.to_string r.repo_url
+         " and points to "^OpamUrl.to_string r.repo_url
        else match r.repo_trust with
-         | None -> "without trust anchors"
+         | None -> " without trust anchors"
          | Some ta ->
-           Printf.sprintf "with trust anchors %s and quorum %d"
+           Printf.sprintf " with trust anchors %s and quorum %d"
              (OpamStd.List.concat_map ~nil:"()" "," String.escaped
                 ta.fingerprints)
              ta.quorum)
+      (OpamRepositoryName.to_string name)
+      (OpamUrl.to_string url)
   | None ->
     let repo = { repo_name = name; repo_url = url;
                  repo_trust = trust_anchors; }

